source/common/common/utility.cc

```diff
@@ -211,15 +211,24 @@ bool DateUtil::timePointValid(MonotonicTime time_point) {
 
 const char StringUtil::WhitespaceChars[] = " \t\f\v\n\r";
 
-bool StringUtil::atoul(const char* str, uint64_t& out, int base) {
+const char* StringUtil::strtoul(const char* str, uint64_t& out, int base) {
```

Comment:
add explicit tests for this new function.

Reply:
OK.